docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>We see that SMC<s>,</s> recognizes this is a harder problem and increase<strong>s</strong> the number of stages. <s>Unfortunatelly </s><strong>Unfortunately</strong> in this case that is not enough<s>t</s> to recover the correct posterior<strong>,</strong> as we can see in the following plot (pay special attention to the rank plot).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>